ArchiSteamFarm/Steam/Integration/SteamChatMessage.cs

```diff
@@ -101,8 +106,17 @@ internal static class SteamChatMessage {
 
 				for (int lineBytesRead = 0; lineBytesRead < lineBytes.Length;) {
 					if (messagePart.Length > prefixLength) {
-						messagePartBytes += NewlineWeight;
-						messagePart.AppendLine();
+						if (messagePartBytes + NewlineWeight + lineBytes.Length > maxMessageBytes) {
```

Comment:
Same here.

Reply:
Not critical, can be done post-PR if somebody has time.